scripts/generate-unicode-data

```diff
@@ -130,15 +130,19 @@ main() {
     # ucd-generate is used to compile regexes into DFAs.
     requires ucd-generate
 
+    mkdir -p src/unicode/fsm/
+
     # For development when ucd-generate needs to be updated.
-    cargo install -f --path /home/andrew/rust/ucd-generate
+    #cargo install -f --path /home/andrew/rust/ucd-generate
```

Comment:
Ah, whoops. My bad on this one. It was nice to have this when I was iterating, but didn't mean to leave it in.